LibreNMS/Authentication/MysqlAuthorizer.php

```diff
@@ -21,6 +21,11 @@ public function authenticate($credentials)
         $password = $credentials['password'] ?? null;
 
         $hash = User::thisAuth()->where(['username' => $username])->value('password');
+        $enabled = User::thisAuth()->where(['username' => $username])->value('enabled');
```

Comment:
you should get both values in one query instead of two mysql queries.

Reply:
combined both requests